### PR TITLE
K season stats

### PR DESCRIPTION
**Pull request by kristastadler**
Added biggest bust and biggest surprise methods Passed spec harness and tests

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
+require 'csv'
```
Comment (brian-greeson) [inline lib/stat_tracker.rb]: Lets Capitalize CSV to get rid of warnings

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -24,6 +24,90 @@ def self.create_items(file, item_class)
...
+        game_data.home_team_id == team_id || game_data.away_team_id == team_id
+      end
+      wins = 0
+      team_games.each_value do |game_data|
+        if team_id == game_data.home_team_id
+          wins += 1 if game_data.home_goals > game_data.away_goals
+        elsif team_id == game_data.away_team_id
+          wins += 1 if game_data.away_goals > game_data.home_goals
+        end
+      end
```
Comment (DavidTTran) [inline lib/stat_tracker.rb:73]: Nested iteration :(

File: test/game_test.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@
 class GameTest < Minitest::Test
 
   def setup
-    StatTracker.create_items("./test/fixtures/games_sample.csv", Game)
+    StatTracker.create_items("./test/fixtures/season_games_sample.csv", Game)
```
Comment (DavidTTran) [inline test/game_test.rb:8]: Truncated information used in game_test.rb

**Review by DavidTTran — APPROVED** [review]
May have to go back over some methods to refactor nested iterations.

**Pull request merged**